content/en/docs/concepts/architecture/nodes.md

```diff
@@ -377,6 +377,21 @@ For example, if `ShutdownGracePeriod=30s`, and
 for gracefully terminating normal pods, and the last 10 seconds would be
 reserved for terminating [critical pods](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical).
 
+{{< note >}}
+When pods were evicted during the graceful node shutdown, they will be marked as failed.
+On `kubectl get pods` you will see the STATUS of these pods being display as `Shutdown`.
```

Comment:
Hi @SergeyKanzhelev . Some questions/suggested changes:

nit: When pods are evicted during the graceful node shutdown, they are marked as failed?
nit: Should STATUS be `Status`?

Is this correct?
Running `kubectl get pod` shows the status of the the evicted pods as `Shutdown`?

nit: And `kubectl describe pod` indicates that the pod was evicted because of node shutdown:

nit: This is a change of behavior compared to abrupt node termination.

Reply:
applied, thank you